test-clients/src/main/java/com/hedera/services/bdd/suites/HapiSuite.java

```diff
@@ -39,7 +39,10 @@
 import org.apache.logging.log4j.Logger;
 
 public abstract class HapiSuite {
-
+    // The first 0 refers to the shard of the target network.
+    // So by hard-coding this, we do add some technical debt we will have to resolve when multiple
+    // shards are deployed.
+    public static final String DEFAULT_SHARD = "0.0.";
```

Comment:
You mean default shard and default realm ?

Reply:
yes tank you